### PR TITLE
Use CoreTests suite in SQL Server Flow and Standalone mode tests

### PR DESCRIPTION
**Pull request by MMatten**
Progress work on resolving issue #23 :-
* Introduce the `CoreTests` suite for StandAlone mode tests.
* Replace `CoreTests.CommonSuite` with full `CoreTests` suite in FlowMode tests.
  * Eliminate duplicate `BulkInsert` test - superseded by `CoreTests.BulkInsert`.

### CODE REVIEW

File: FitNesseRoot/DbFit/AcceptanceTests/DotNetTests/SqlServerTests/FlowMode/BulkInsertWithReturning/content.txt [hunk 1]
```diff
@@ -1,14 +1,13 @@
-|execute| delete from Users|
+|execute| delete from users|
```
Comment (javornikolov) [inline FitNesseRoot/DbFit/AcceptanceTests/DotNetTests/SqlServerTests/FlowMode/BulkInsertWithReturning/content.txt]: May we slice all reformatting to separate commit?

**Comment by javornikolov** [comment]
@MMatten, at first glimpse the formatting changes like `users -> Users` are not something essential for this PR. Can we slice them out to separate commit and maybe separate PR? (This would make the commit with the main change simpler and more obvious).
The rest looks OK to me.

**Comment by MMatten** [comment]
@javornikolov

The object name case change is required to use the CoreTests suite as the SQL Server test DB is configured to use a case sensitive collation. 

The other changes (not the object name case) to the DB setup script are to make it run properly end to end. I could move these fixes out if you think that's better.

**Comment by javornikolov** [comment]
OK, I understand now - seems reasonable to keep these changes together. :+1:
I'm merging the PR.

**Pull request merged**